src/cpp/server/extras/conf/database.conf

```diff
@@ -0,0 +1,34 @@
+# This file contains database configuration.
+#
+# RStudio Server needs an external database for storage of specific configuration and runtime information.
+# At present, SQLite and PostgreSQL databases are supported.
```

Comment:
I'd remove this and just ask users to reference the admin guide, as this could become stale as new databases are supported.

Reply:
I'm not overly worried about adding support for new databases and this becoming stale. We match feature parity with Connect, which has only had support for SQLite and PostgreSQL for some time now. I think at best, we'd add support for ODBC (which essentially adds support for all databases).

Also, it is common to provide such easy-to-change configurations for pro installations, so figured we'd just add this here since we will likely want an install for pro.

Reply:
I think I misunderstood your comment - I think you were suggesting we just take out line 4? I can make that change!